changes/add_GLT-3650.md

```diff
@@ -0,0 +1 @@
+added notes field to Worksets
```

Comment:
Do changelogs starting with 'add_' need 'Added' in the message? Don't they compile under the Added: header?

Reply:
ah right, forgot about that.